app/views/scheduled_messages/_form.html.erb

```diff
@@ -5,6 +5,7 @@
     hint: "Don't forget, you can format your message using #{link_to "Slack's message formatting rules",
     "https://slack.zendesk.com/hc/en-us/articles/202288908-Formatting-your-messages", target: "_blank"}.".html_safe %>
   <%= form.input :days_after_start, label: "Days after employee starts to send message" %>
+  <%= form.input :time_of_day, label: "Time of day to send message (respects employee's local time zone)", ampm: true, input_html: {class: "date-time-select"} %>
```

Comment:
Thanks!  "Uses" does sound better, will change to that or something similar. :-)